Tests/Tests/NSDictionary+StripeTest.swift

```diff
@@ -53,43 +53,43 @@ class NSDictionary_StripeTest: XCTestCase {
                         "id": "fee_123"
                     ], ["payment"],
                 ],
-            ] as NSDictionary
+            ] as [AnyHashable : Any]
 
-        let result = dictionary.stp_dictionaryByRemovingNulls() as NSDictionary
-        XCTAssertEqual(result, expected)
+        let result = dictionary.stp_dictionaryByRemovingNulls()
+        XCTAssertEqual(result as NSDictionary, expected as NSDictionary)
     }
 
     func test_dictionaryByRemovingNullsValidatingRequiredFields_keepsEmptyLeaves() {
         let dictionary =
             [
                 "id": NSNull()
-            ] as NSDictionary
-        let result = dictionary.stp_dictionaryByRemovingNulls() as NSDictionary
+            ] as [AnyHashable: Any]
+        let result = dictionary.stp_dictionaryByRemovingNulls()
 
-        XCTAssertEqual(result, [:])
+        XCTAssertEqual(result as NSDictionary, [:] as NSDictionary)
     }
 
     // MARK: - dictionaryByRemovingNonStrings
     func test_dictionaryByRemovingNonStrings_basicCases() {
         // Empty dictionary
-        var dictionary = [:] as NSDictionary
-        var expected = [:] as NSDictionary
-        var result = dictionary.stp_dictionaryByRemovingNonStrings() as NSDictionary
-        XCTAssertEqual(result, expected)
+        var dictionary = [:] as [AnyHashable: Any]
+        var expected = [:] as [AnyHashable: Any]
+        var result = dictionary.stp_dictionaryByRemovingNonStrings()
+        XCTAssertEqual(result as NSDictionary, expected as NSDictionary)
 
         // Regular case
         dictionary =
             [
                 "user": "user_123",
                 "nicknames": "John, Johnny",
-            ] as NSDictionary
+            ]
         expected =
             [
                 "user": "user_123",
                 "nicknames": "John, Johnny",
-            ] as NSDictionary
-        result = dictionary.stp_dictionaryByRemovingNonStrings() as NSDictionary
-        XCTAssertEqual(result, expected)
+            ]
+        result = dictionary.stp_dictionaryByRemovingNonStrings()
```

Comment:
Looks like this one is unused now fwiw. Seems like it was used for `metadata`, which is no longer sent down for publishable keys.